src/pages/steps/ScopeStep.tsx

```diff
@@ -10,7 +10,7 @@ import { ToggleBox } from '@/ui/ToggleBox';
 
 import { FormDataIntersection, StepProps } from './Steps';
 
-const TypeStep = <TFormData extends FormDataIntersection>({
+const ScopeStep = <TFormData extends FormDataIntersection>({
```

Comment:
I'm still struggling with the name `scope` 😅  but other than that it looks good 👌